pkg/cli/environment.go

```diff
@@ -56,6 +59,8 @@ type EnvSettings struct {
 	RepositoryCache string
 	// PluginsDirectory is the path to the plugins directory.
 	PluginsDirectory string
+	// MaxHistory is the name of the env var for setting max history.
```

Comment:
MaxHistory is the name of the env var for setting max history. --> MaxHistory is the max release history maintained.